ios/RCCViewController.m

```diff
@@ -351,6 +351,19 @@ - (void)updateStyle
   [self setStyleOnAppearForViewController:self appeared:true];
 }
 
+- (BOOL) isViewLoaded {
```

Comment:
Overriding `isViewLoaded` here to have a side-effect doesn't seem like a safe idea. Access to this property is not guaranteed.

Can you set a breakpoint in this method and figure out what lifecycle we should be hooking into (aka, the one that's calling this)?

It looks like we run this logic already on `viewWillAppear` and `viewDidAppear`, but perhaps we need to call this on `viewDidLoad` as well.

Reply:
agree, I tried different techniques, but this the only one worked for me. 
But I'll take a look a bit more...